src/store/saga/UserInfoForm/UserInfoForm.ts

```diff
@@ -0,0 +1,31 @@
+import { ADD_DATA } from "store/types";
+import { call, takeLatest } from "redux-saga/effects";
+import axios from "axios";
+
+interface ResponseGenerator {
+  config?: any;
+  data?: any;
```

Comment:
not all the parameters should be optional in an interface like this one!!

Reply:
ok